tests/functional/data/example/pyproject.toml

```diff
@@ -19,3 +19,12 @@ pygments = ["pygments"]
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[[tool.poetry.source]]
+name = "pypi-all-source1"
+url = "https://pypi.python.org/simple"
+default = true
+
+[[tool.poetry.source]]
+name = "pypi-all-source2"
+url = "https://pypi.python.org/all"
```

Comment:
`poetry export` is not actually including this as an extra index, but that seems like an upstream issue.